### ISSUE TITLE
Transaction subscriber should also returns error

### ISSUE THREAD

**Issue by Neylix** [archethic-foundation/archethic-node#445]
### Is your feature request related to a problem?

The problem with TransactionSubscriber  is that it notify only the validated transaction (from the replication attestation).
But if the transaction is invalid for any kind of reason, the TransactionSubscriber is not notified and so the user will wait until timeout for the transaction validation.

### Describe the solution you'd like

We should provide a way to notify the welcome node when a transaction is invalid.

- [x] Understand the distributed workflow process
- [x] Understand when the welcome node is notified for a validated transaction
- [x] Search a solution to notify welcome node in any case when a transaction is invalid
- [x] Implement it

**Comment by Neylix** [comment]
For exemple I got an error when trying to create a smart contract, this error should be catched an returned throught the subscriber  to inform the user about the error

![image](https://user-images.githubusercontent.com/26260538/178855917-2493094f-d20b-40d1-b618-ec0e15ac6619.png)

**Comment by imnik11** [comment]
How can one reproduce this ?

**Comment by Neylix** [comment]
> How can one reproduce this ?

You can create a smart contract with this code : 
```
actions triggered_by: interval, at: "0 0 0 0 2" do
    set_type transfer
    add_uco_transfer to: "000066f7dc977185b026a4792a9415c0b460880f6fbffe876d9f308c930d0f8daaa1", amount: 100000000
    end
```

**Comment by internet-zero** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Product-and-Design-Workspace-61efe71b9083a90018250d89/issues/archethic-foundation/archethic-node/445?planning-poker) with ZenHub @apoorv-2204 @imnik11 @Neylix @prix-uniris

**Comment by samuel-uniris** [comment]
Closed by #475
